common/persistence/sql/sqlplugin/matching_task_queue.go

```diff
@@ -64,6 +64,25 @@ type (
 		DataEncoding  string
 	}
 
+	AddBuildIdToTaskQueueMapping struct {
+		NamespaceID   []byte
+		TaskQueueName string
+		BuildIds      []string
+	}
+
+	RemoveBuildIdToTaskQueueMapping struct {
+		NamespaceID   []byte
+		TaskQueueName string
+		BuildIds      []string
+	}
+
+	GetTaskQueuesByBuildIdRequest struct {
+		NamespaceID []byte
+		BuildID     string
+	}
+
+	CountTaskQueuesByBuildIdRequest = GetTaskQueuesByBuildIdRequest
```

Comment:
I will, just curious why

Reply:
otherwise you could pass a value of one type as the other? I mean, yeah, it's not like it's a big problem, they are the same in some sense, it's just weird to be able to pass a `*CountTaskQueuesByBuildIdRequest` to `GetTaskQueuesByBuildId`